automation/test-execution/ansible/scripts/mlflow-quick-log.sh

```
@@ -0,0 +1,223 @@
+            -e "mlflow_tracking_uri=${MLFLOW_URI}" \
+            ${EXTRA_ARGS}
+        ;;
+
+    today)
+        echo -e "\n${YELLOW}Importing results from today...${NC}"
+        COUNT=0
+
+        while IFS= read -r bench; do
+            meta="${bench%/*}/test-metadata.json"
+            if [ -f "$meta" ]; then
+                echo -e "Logging: ${GREEN}$(basename $(dirname "$bench"))${NC}"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)